jwst/outlier_detection/outlier_detection.py

```diff
@@ -158,108 +173,106 @@ def do_detection(self):
 
         # Perform outlier detection using statistical comparisons between
         # each original input image and its blotted version of the median image
-        detect_outliers(self.input_models, blot_models,
-            self.reffiles, **self.outlierpars)
+        self.detect_outliers(blot_models)
```

Comment:
If `resample_data` was not true (e.g. when you're processing coronagraphic or TSO data), will `blot_models` contain the unresampled data?

Reply:
The previous code already accommodates the case where resampling is done (default imaging case) and when it is NOT done (as needed for TSO or coron data).  It will actually contain the unresampled data as copies of the median model.